functions/Find-DbaOrphanedFile.ps1

```diff
@@ -0,0 +1,225 @@
+﻿
+Function Find-DbaOrphanedFile
+{
+<#
+.SYNOPSIS 
+Find-DbaOrphanedFile moves/removes orphaned database files
+
+.DESCRIPTION
+Get all the database files for all the database for the instance
+Get the various directories of the instance and get all the present database files.
+Compare which the two lists to see if there are any orphaned files and return the list
+

```

Comment:
Would reword this to be "find" instead of "get", to just match the verb used in naming the command. 😉 

Would also recommend formatting this in 2-3 sentence paragraph, just looks like bullet list. Description does not have to explain exactly what is done, more of what problem it solves.